Gemfile.lock

```diff
@@ -1,62 +1,60 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    better_errors (2.1.1)
+    better_errors (2.5.0)
       coderay (>= 1.0.0)
-      erubis (>= 2.6.6)
+      erubi (>= 1.0.0)
       rack (>= 0.9.0)
-    byebug (9.0.6)
-    coderay (1.1.1)
-    commander (4.4.3)
-      highline (~> 1.7.2)
-    database_cleaner (1.6.1)
+    byebug (10.0.2)
+    coderay (1.1.2)
+    commander (4.4.7)
+      highline (~> 2.0.0)
+    database_cleaner (1.7.0)
     diff-lcs (1.3)
-    erubis (2.7.0)
-    event_sourcery (0.13.0)
-    event_sourcery-postgres (0.3.0)
-      event_sourcery (>= 0.10.0)
+    erubi (1.7.1)
+    event_sourcery (0.22.0)
+    event_sourcery-postgres (0.8.0)
+      event_sourcery (>= 0.14.0)
       pg
-      sequel (~> 4.38)
-    highline (1.7.8)
-    method_source (0.8.2)
-    mustermann (1.0.0)
+      sequel (>= 4.38)
+    highline (2.0.0)
+    method_source (0.9.2)
+    mustermann (1.0.3)
     pg (0.20.0)
-    pry (0.10.4)
+    pry (0.12.2)
       coderay (~> 1.1.0)
-      method_source (~> 0.8.1)
-      slop (~> 3.4)
-    pry-byebug (3.4.2)
-      byebug (~> 9.0)
+      method_source (~> 0.9.0)
+    pry-byebug (3.6.0)
+      byebug (~> 10.0)
       pry (~> 0.10)
-    rack (2.0.3)
-    rack-protection (2.0.0)
+    rack (2.0.6)
+    rack-protection (2.0.4)
       rack
-    rack-test (0.6.3)
-      rack (>= 1.0)
-    rake (12.0.0)
-    rspec (3.6.0)
-      rspec-core (~> 3.6.0)
-      rspec-expectations (~> 3.6.0)
-      rspec-mocks (~> 3.6.0)
-    rspec-core (3.6.0)
-      rspec-support (~> 3.6.0)
-    rspec-expectations (3.6.0)
+    rack-test (1.1.0)
+      rack (>= 1.0, < 3)
+    rake (12.3.1)
```

Comment:
### [rake](https://github.com/ruby/rake)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 12.0.0 → 12.3.1

[_~~change-log~~, [source-code](https://github.com/ruby/rake)_]





<details>
<summary>Commits</summary>

A change of **145** commits. See the full changes on [the compare page](https://github.com/ruby/rake/compare/v12.0.0...v12.3.1).

These are the first 10 commits:
- (3d6665c) [test for chained extensions in rules](https://github.com/ruby/rake/commit/3d6665c7d7fd902fc91e40dcf9a0993009833a68)
- (84bc674) [support chained extensions in rules](https://github.com/ruby/rake/commit/84bc6741da1a6c4b76267e3f11fdff602879fe81)
- (71f02c9) [Merge remote-tracking branch 'upstream' into chained_extensi…](https://github.com/ruby/rake/commit/71f02c99267afd9cfec2e9aee72c0c886bbd6efc)
- (24cafd5) [cherry-picked 65b643209d82e7f04f5568aa1c22d3969fef3f2a](https://github.com/ruby/rake/commit/24cafd5a48096bb526a1180d7067b472f412fc50)
- (050c7d4) [History](https://github.com/ruby/rake/commit/050c7d4900a691215e953b1de122d1bd60d32254)
- (0857636) [bump version to 11.3.0](https://github.com/ruby/rake/commit/08576365d9acb9a76ce36a4448953be21f001b5f)
- (2a15e60) [up to date](https://github.com/ruby/rake/commit/2a15e60f3437f494754097523318af8360c5ccf9)
- (477535e) [workaround for bundler-1.13.1](https://github.com/ruby/rake/commit/477535e2e11386ae661a0a881d331f735713d48c)
- (083267f) [Fixed grammers](https://github.com/ruby/rake/commit/083267f77dbaff199460f6fd11ef6e8ffeca7643)
- (8524523) [Fixes documentation typo in rakefile.rdoc](https://github.com/ruby/rake/commit/8524523cf93682be9f717d5ae364dbb36ee6c3d3)


</details>